assets/src/edit-story/output/page.js

```diff
@@ -61,6 +69,12 @@ function OutputPage({ page }) {
             <OutputElement key={'el-' + element.id} element={element} />
           ))}
         </div>
+        {backgroundOverlay && backgroundOverlay !== 'none' && (
```

Comment:
Is this expected to only apply to the safe area? I'd assume it should go to the fullbleed at all times, or at least would match the background.

Reply:
Hmm it does, were you looking at an older commit?